src/gtk/spinctrl.cpp

```diff
@@ -523,4 +661,28 @@ void wxSpinCtrlDouble::SetDigits(unsigned digits)
     GtkSetEntryWidth();
 }
 
+wxSpinCtrl::GTKInputResult wxSpinCtrlDouble::GTKInput(double* value) const
+{
+    return wxSpinCtrlGTKBase::GTKInput(value);
+}
+
+bool wxSpinCtrlDouble::GTKOutput(wxString* text) const
+{
+    if ( wxSpinCtrlGTKBase::GTKOutput(text) )
```

Comment:
There doesn't seem to be much point in making `GTKOutput()` pure virtual when one of the overrides just calls the base. Unless you're planning something more here later. Could be done more simply as just
```suggestion
    return wxSpinCtrlGTKBase::GTKOutput(text);
```

Reply:
Sorry, I should have mentioned it, but I do plan to do more in it, the "new UI locale API" thread on wx-dev -- basically I'm going to convert between strings and doubles manually here soon (BTW, if you have any thoughts on feasibility of the solution proposed in this thread for wxGTK, I'd love to hear them).